### DISCUSSION TITLE
Linearmouse Exact sensitivity for Windows 6/11?

### DISCUSSION THREAD

**Discussion by siraidenkim** [linearmouse/linearmouse#500] [Q&A]
Hey guys, I have a Razer essential that I use at 800 dpi/500 hz with Linearmouse. I'm recently transitioning from MAC to Windows, and was wondering what the exact linear mouse sensitivity I should have for Windows 6-11 tracking speed. Here is some of my mac settings: 

My mouse settings: tracking (lowest), DPI (800), Linearmouse accel off

By the way, I read this discussion, but was extremely confused: https://github.com/linearmouse/linearmouse/discussions/383#discussion-4920276 

I also read this one, https://github.com/linearmouse/linearmouse/discussions/121#discussion-4057399, but I dont currently have my pc with me as it broke down, so it is of no help.

**Comment by HelveticaStandardBold** [comment]
I would also like to see if it would be possible to get an exact sensitivity conversion, or at the very least a calculator that you could input your windows settings and find the exact settings to emulate that on macOS.

**Reply by RobaloAmigo** [reply]
+1 and same for Logitech GHUB DPI options

**Reply by Hoe2020** [reply]
I made an enhancement request for that.

**Comment by HUMORCE** [comment]
Really? eDPI of a mouse that 800DPI with lowest tracking speed + acceleration disabled on macOS may lower than 200.

**Comment by Enjoy2KOW** [comment]
Would like to know what's the equivalent 6/11 windows to macOS.
I've put this on linearmouse : 
![image](https://github.com/user-attachments/assets/43c7e467-ca08-493a-80d4-d58f4ff23d4b)

**Reply by ryanmortier** [reply]
I did a lot of testing and it appears that 1 is very close if not exact to 6/11 in windows.

I used the DPI analyzer here: https://www.mouse-sensitivity.com/dpianalyzer/

**Comment by wasabi25** [comment]
Hello, I've learned using 0.01 acc, and then choosing the speed after, the closest to Windows.

**Comment by angaylo22** [comment]
it should be set to 6/11 by default when you switch the "Disable pointer acceleration" feature on for the first time with the mouse
![image](https://github.com/user-attachments/assets/6538f1ea-0539-4efc-823c-f409ec1b864f)

I verified this by comparing it inside of games and it being identical to my windows dpi using a ruler

I also used this tool a year ago to test it and it seemed accurate enough
https://www.mouse-sensitivity.com/dpianalyzer/

I may not be able to reconfirm my findings until I receive my new mouse because the mouse I last tested it with (Deathadder v3) broke

**Comment by RobaloAmigo** [comment]
I personally use 0,6875 for pointer acceleration and 0,069 for tracking speed. Pointer is fast and responsive but still under control. Works well both for trackpad and mouse for me.

**Comment by vadim9687** [comment]
It’s not possible because windows has 3 acceleration points ( 2 break points ) while Mac has 2 acceleration points ( 1 break point )

**Reply by Enjoy2KOW** [reply]
So what do you recommend for being the same as 6/11 on Windows?

**Reply by vadim9687** [reply]
I currently use speed 0.04 and accel .5, it's closer than MacOs's default settings but still not as natural as Windows' EPP
